LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2LWAbilitiesModTemplate.uc

```diff
@@ -234,6 +236,14 @@ static function GetUpdatedHitChances(X2AbilityToHitCalc_StandardAim ToHitCalc, o
 	}
 }
 
+static function UpdateTeamwork(X2AbilityTemplate Template)
+{
+	Template.AbilityCosts[0].bConsumeAllPoints = true;
```

Comment:
This doesn't compile. You need to cast the ability cost to `X2AbilityCost_ActionPoints` before you can set this property.

Reply:
Hm, are you sure that doesn't compile because of that, and not because Abilitycosts is protectedwrite in non-highlander code?

Reply:
`AbilityCosts` is not `protectedwrite`.